src/main/java/frc/robot/subsystems/swerve/Swerve.java

```diff
@@ -42,25 +43,21 @@ public class Swerve extends GBSubsystem {
     private final SwerveState currentState;
     private Supplier<Rotation2d> currentAngleSupplier;
 
-    public Swerve() {
+    public Swerve(RobotType robotType) {
         setName(getClass().getSimpleName());
```

Comment:
Consider moving this to super

Reply:
maybe there will be subsystems like pose estimator stuff that don't need it

Reply:
Every subsystem needs a name.

https://youtu.be/1Tj0tSlU2Xg?si=DU7CJlM8T3DyzSIY